frontend/src/app/submitted-topic-drafts/submitted-topic-draft-action-button/submitted-topic-draft-action-button.component.ts

```diff
@@ -42,86 +43,58 @@ export class SubmittedTopicDraftActionButtonComponent implements OnDestroy, OnIn
   withDrawApprovalTopicDraftText: string;
   rejectTopicDraftText: string;
   withDrawRejectionTopicDraftText: string;
-  submitTopicDraftText: string;
-  withDrawSubmitTopicDraftText: string;
-  adminOrInitiatorTooltip: string;
-  statusMustBeSubmitted: string;
-  statusMustBeSubmittedAndUser: string;
+  notAdminToolTip: string;
+  notApprovedToolTip: string;
 
   constructor(private topicDraftMapper: TopicDraftMapper,
               private currentUserService: CurrentUserService,
               private translate: TranslateService,
               private dialog: MatDialog,
-              private snackBar: MatSnackBar) {
-  }
+              private router: Router
+              ) {}
 
   ngOnDestroy(): void {
     this.subscriptions.forEach(sub => sub.unsubscribe());
     this.subscriptions = [];
   }
 
   ngOnInit(): void {
-    this.subscriptions.push(this.translate.stream('submitted-topic-draft-action-button.edit-tooltip.status')
-      .subscribe((text: string) => {
+    this.translate.stream('submitted-topic-draft-action-button.edit-tooltip.status').subscribe((text: string) => {
```

Comment:
In dieser Datei sind sehr viele subscriptions, die nicht unsubscribed werden. Vielleicht über ein subscription array lösen.